_posts/blog/2019-06-20-Hello!I am Yini He.md

```diff
@@ -14,63 +14,90 @@ author: Yini-He
 </figure>
                   
-At the same time, I am interested in all kinds of traditional Chinese culture, with its food, traditional clothing, calligaraphy, tea and Guqin(an ancient instrument of China). With a history of 5000 years, China has preserved its ancient philosophy and beautiful civilization.  More often, I am fascinated by the merging of Chinese and western cultures. I used to play the ukelele wearing Qipao(a traditional dress of China).  I also tried cooking a Chinese meal with fish and chips. 
+At the same time, I am interested in all kinds of traditional Chinese culture, with its food, traditional clothing, calligaraphy, tea and Guqin(an ancient instrument of China). With a history of 5000 years, China has preserved its ancient philosophy and beautiful civilization. I am fascinated by the merging of Chinese and western cultures. I used to play the ukelele wearing Qipao(a traditional dress of China).  I also tried cooking a Chinese meal with fish and chips. 
 
 <figure>
-  <img src="/images/YiniHe/3qipao3.jpg">
+  <img src="/images/YiniHe/3qipao5.jpg">
+  <figcaption> One of the most beautiful nouns in the world is culture(photo by Yini) </figcaption>
 </figure>
 
-# Life and work goals
+## Life and work goals
 
---Ideal comes from life（理想源于生活）
++>Ideal comes from life（理想源于生活）
 
-My university major is psychology.  In my senior year, I decided to go to a special school and work as an intern teacher there.  The students at this school were suffering from mental disorder. This was the first time that I truly understood the importance of mental health at heart.  I love those children even when some of them say things that are illogical and incoherent, while still others just always sat in a corner. But there are those who have great artistic talents.  Some of them are attractive and have a friendly disposition, willing to be best friends with me. Ever since then, I’ve always thought about these teenagers and hope that they could have a better life with the help of medical and therapeutic advancements for mental health. 
+My university major is psychology.  
+In my senior year, I decided to go to a special school and work as an intern teacher there.  
+The students at this school had mental health differences. 
+This was the first time that I truly understood the importance of mental health at heart.  
+I loved those children. Many of them had great artistic talents.  
+Some of them had attractive and friendly dispositions, and were willing to be best friends with me. 
+At the same time I was distressed to think that they could face social difficulties and social burdens. 
+Ever since then, I’ve always thought about these teenagers and hoped that they could have a better life with the help of medical and therapeutic advancements for mental health. 
 
 <figure>
-  <img src="/images/YiniHe/4yuanmingyuan3.jpg">
+  <img src="/images/YiniHe/4yuanmingyuan!_5.jpg">
```

Comment:
Can you save this file name without the `!_`? Just rename to `4yuanmingyuan5.jpg` because it makes the formatting in markdown rather strange to have the `_` and the `!` in the file name.

Reply:
Thank you for reminding me this good  habit.